### ISSUE TITLE
Unable to remove myself from a project

### ISSUE THREAD

**Issue by bertjwregeer** [pypa/warehouse#6365]
**Describe the bug**

I would like to remove myself as owner/maintainer of a project on PyPI, but I am unable to do so using the web interface (not sure if the API allows this or not).

This means that someone can add me to a project, which then shows up on my page of projects I maintain (https://pypi.org/user/xistence/) without my consent and without the ability to remove myself from said project.

As an owner on a project, I can't remove myself:
![Screen Shot 2019-08-05 at 13 04 35](https://user-images.githubusercontent.com/649426/62488515-af3a2b80-b781-11e9-9fc8-c1c2b2983fec.png)

If I am a lonely maintainer, I can push new releases but can't manage the project at all:

![Screen Shot 2019-08-05 at 13 04 43](https://user-images.githubusercontent.com/649426/62488521-b19c8580-b781-11e9-9bda-fbfac4b7e144.png)

**Expected behavior**

To have the ability to manage which projects I am a maintainer/owner of by removing myself from a project if necessary.

**To Reproduce**

- Have someone add you as a second maintainer/owner on a project
- Attempt to figure out how to remove yourself from said project

**My Platform**

If you believe the platform I am on affects this, please let me know and I'll get back to you as soon as possible with this information... I hope this isn't the case though and it's not feature flagged to only work on Seamonkey version 232.23942532 and not on Safari 23129542.23423 or Chrome 2043254295432954254295432 :-P

**Additional context**

<3

**Comment by di** [comment]
The ability to add users to a project w/o consent is #5790 and should be resolved by #6206.

Once we add that feature, we can add the ability to remove yourself from a project (as long as you're not the sole owner), but for now we want to avoid the situation where someone creates a project, adds another user as owner, and then can remove themselves.

> If I am a lonely maintainer, I can push new releases but can't manage the project at all:

This is expected behavior for the "maintainer" role: maintainers can only upload new releases.

**Comment by mmerickel** [comment]
Invitations are good but #5790 doesn't cover the primary issue which is that a maintainer legitimately just wants to leave a project. They should not need to ask a project owner to remove them from the project after they have joined it via invite or otherwise.

**Comment by di** [comment]
@mmerickel Yes, that's why I've left this issue open and added the "feature request" label, but this is blocked on fixing #5790 first.

**Comment by mmerickel** [comment]
@di thanks for clarifying. I misread your original comment as stating that maintainers should only be uploading releases and unable to remove themselves.

**Comment by bertjwregeer** [comment]
@di if someone adds me or anyone else as an owner and then removes themselves, I should still be able to remove myself from the project. Sending the project to the graveyard, potentially allowing the name to be reclaimed, or leaving it in limbo.

Or at least having some way to remove it from the public listing of projects tied to my name would be a great start.

I still think maintainers should be able to view the "management" part of the project, just maybe not allow them to make edits. Right now to see who else is a maintainer/owner on the project I have to click the view button, which drops me back to the public pages, outside of the management interface. It feels a little disjointed.

Either way, I opened this mainly because I left a company today and I wanted to do the right thing and leave the projects that I was no longer going to be maintaining, but found it to not be possible. I look forward to #5790 and then this being fixed :-)

**Comment by bertjwregeer** [comment]
Looks like I accidentally filed a duplicate of #4729. Linking to that and closing this ticket. Thanks for the feedback @di
